library/macros/bitbucket/pr-trigger.yaml

```diff
@@ -0,0 +1,40 @@
+- trigger:
+    # Base trigger for pull-requests
+    #
+    # Poll BitBucket server and trigger job on
+    # newly created requests or changed ones.
+    #
+    # This is shortened version of original trigger with recommended
+    # default values.
+    #
+    # Parameters:
+    #
+    # timer           how often to poll server in cron fmt,
+    # base-url        base url of your server, fe http://git.my.com/
+    # project         name of project (short key) to check
+    # repo            name of repo (slug) to check
+    # branch          name of branch to check
+    # credentials-id  credentials from Jenkins credentials to use for
+    #                 polling BitBucket API (login+password)
+
+    name: library/bitbucket/pr-trigger
+
+    triggers:
+    - stash:
+        spec: '{timer}'
+        cron: '{timer}'
+        stash_host: '{web-url}'
+        project_code: '{project}'
+        repository_name: '{repo}'
+        credentials-id: '{credentials-id}'
+        ci_skip_phrases: 'NO TEST'
+        ci_build_phrases: 'test this please'
+        target_branches_to_build: '{branch}'
+        ignore_ssl: 'true'
```

Comment:
in published ignore_ssl is true, here is false.
it would be better if this parameter will be also configurable.